examples/jms-producer-transaction/jms_producer_transaction.curl.out

```diff
@@ -0,0 +1 @@
+$ curl http://localhost:9090/orders -H "Content-type:application/json" -d "{\"orderId\": 1, \"productName\": \"Sport shoe\", \"price\": 27.5, \"isValid\": true}"
```

Comment:
here also we can remove escape slashes

Reply:
Fixed with [161c5208951e36aa7627a84e7d00be7c30f0995d](https://github.com/ballerina-platform/ballerina-distribution/pull/4880/commits/161c5208951e36aa7627a84e7d00be7c30f0995d)